### PR TITLE
books/2015/pawga: chapter 2/17, section 9/22

### PR DESCRIPTION
**Pull request by gaverhae**

### CODE REVIEW

File: books/2015/pro-android-web-game-apps-2012/summary/Chapter2.md [hunk 1]
```diff
@@ -103,13 +103,91 @@ draw an outline (`drawRect`) or a filled shape (`fillRect`). They both take
...
+ctx.stroke(); // this makes the black line from 50, 50 to 120, 100 appear
+```
+
+We use this to draw a grid in [listing 2.5]:
+
+```javascript
+      var cellSize = 40;
+      ctx.beginPath();
+
+      for (var i = 8; i < 8; i++) {
```
Comment (JustineSara) [inline books/2015/pro-android-web-game-apps-2012/summary/Chapter2.md:135]:
```suggestion
      for (var i = 0; i < 8; i++) {
```

File: books/2015/pro-android-web-game-apps-2012/summary/Chapter2.md [hunk 2]
```diff
@@ -103,13 +103,91 @@ draw an outline (`drawRect`) or a filled shape (`fillRect`). They both take
...
+        ctx.moveTo(i * cellSize + 0.5, 0);
+        ctx.lineTo(i * cellSize + 0.5, cellSize * 6);
```
Comment (JustineSara) [inline books/2015/pro-android-web-game-apps-2012/summary/Chapter2.md:137]: if I get that correctly those are the 8 vertical lines that creates 7 columns for "height" of 6 rows (which we define below)

**Review by JustineSara — APPROVED** [review]

File: books/2015/pro-android-web-game-apps-2012/summary/Chapter2.md (hunk 2 above)
Comment (gaverhae) [reply books/2015/pro-android-web-game-apps-2012/summary/Chapter2.md:137]: Indeed!

File: books/2015/pro-android-web-game-apps-2012/summary/Chapter2.md (hunk 1 above)
Comment (gaverhae) [reply books/2015/pro-android-web-game-apps-2012/summary/Chapter2.md:135]: 🤦

I corrected it in the listing, but forgot I'd also put a copy in the text. Thanks for catching it!

**Pull request merged**